### DISCUSSION TITLE
Visual Studio Code

### DISCUSSION THREAD

**Discussion by dbeaver12** [microsoft/vscode-discussions#658] [Extension Development QnA] [answered]
Hi,
With ref. to a ticket raised on MMU system(71403360) we would like to know a few questions prior to its use in one of our departments in Manchester Metropolitan University –
1. When was software last updated
2. Is software still on active development
3. Are the software provider committed to patching the software in event of security incident or known security vulnerabilities?
We would also like to know a list of queries for our Info sec team regarding Visual Studio Code.

Thanks ,

Jesu

Jesu Mathew 
Software Licensing Administrator

**Comment by gjsjohnmurray** [comment, marked as answer]
This is not an appropriate place to ask. Please review the information in the post pinned at the top left of https://github.com/microsoft/vscode-discussions/discussions
